CHANGELOG.md

```diff
@@ -112,7 +112,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### API Breaking Changes
 
-* !(storev2alpha1) [#13370](https://github.com/cosmos/cosmos-sdk/pull/13370) remove storev2alpha1
+* (!storev2alpha1) [#13371](https://github.com/cosmos/cosmos-sdk/pull/13371) Remove storev2alpha1 from cosmos-sdk
```

Comment:
Whats with the `!`?

Reply:
thought those were for removals?

Reply:
They are for breaking changes :) but necessary only in the PR / commit title